Product/Production/Gateway/CONNECTGatewayWeb/src/main/java/gov/hhs/fha/nhinc/gateway/servlet/InitServlet.java

```diff
@@ -63,8 +62,10 @@ public void init(ServletConfig config) throws ServletException {
         LOG.debug("InitServlet start...");
         executor = Executors.newFixedThreadPool(ExecutorServiceHelper.getInstance().getExecutorPoolSize());
         largeJobExecutor = Executors.newFixedThreadPool(ExecutorServiceHelper.getInstance()
-                .getLargeJobExecutorPoolSize());
-
+            .getLargeJobExecutorPoolSize());
+        //Initialize HibernateUtil when CONNECTGatewayWeb is initialized required for AuditRepo JavaImpl EJB calls.
+        //Do not Remove this ...
+        SessionFactory session = HibernateUtil.getSessionFactory();
```

Comment:
Do we need the local variable session anywhere?  Otherwise, just call getSessionFactory without assigning it to anything.